Misc/NEWS.d/next/Core_and_Builtins/2025-02-22-01-23-23.gh-issue-130425.x5SNQ8.rst

```diff
@@ -0,0 +1,2 @@
+Add ``"Did you mean: 'attr'?"`` suggestion when using ``del obj.attr`` if ``attr``
```

Comment:
Let's also add a `What's New` entry as this is a user-facing change. For that you need to update `Doc/whatsnew/3.15.rst`. I don't know if there is a built-in section or language change section already. If not, look at 3.14.rst where we indicated that we improved error messages.